### PR TITLE
doc: update erasure-code-profile.rst

### PR DESCRIPTION
**Pull request by guits**
Update this part of the documentation to reflect the change made by
3e86be7d50d485dbcce50d99d2f52891ca7bd399

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test docs

**Comment by guits** [comment]
jenkins test render docs

File: doc/rados/operations/erasure-code-profile.rst [hunk 1]
```diff
@@ -10,8 +10,8 @@ erasure coded pool and the associated CRUSH rule.
 The **default** erasure code profile (which is created when the Ceph
 cluster is initialized) provides the same level of redundancy as two
 copies but requires 25% less disk space. It is described as a profile
```
Comment (tchaikov) [inline doc/rados/operations/erasure-code-profile.rst]: > provides the same level of redundancy as two copies but requires 25% less disk space.


could you help update these numbers as well? as they are not accurate anymore. if k=2, m=2, we will split the data into 2 equal-sized chunks, and have 2 parity chunks of the same size. so, a file sizes 4K will be stored in cluster as 4 chunks, each chunk sizes 2k. so, in this case:
```
2k*2 +2k*2 = 8k
```
it's identical to the overhead of a 2-replica pool, where we also needs 8k space for storing a 4k chunk (without taking the overhead of the metadata and logging etc into consideration).

while when it comes to "level of redundancy", a pool using this erasure profile can sustain the data loss of 2 chunks out of 4 at a given time. while a replicated pool of 2 copies can only withstand 1 replica of data loss out of 2. if we use the possibility of of data availability of as the metric of "level of redundancy", then the redundancy offered by k=2,m=2 is , where p is the possibility of failure of a disk, k = 2, and m = 2

while the possibility of of data availability of a 2-replica pool is  .

so, in short, k=2,m=2 erasure provides better data availability than pool size =2 replicated pool.
